misk/web/@misk/components/src/components/TextHTMLOrElementComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
+
+/**
+ * <TextHTMLOrElementComponent content={<div>item</div>} length={35} />
```

Comment:
Is there a reason to not use the element's children?

Reply:
Changed it to use children 👍